Rubberduck.SmartIndenter/IndenterSettings.cs

```diff
@@ -219,45 +221,52 @@ public void LoadLegacyFromRegistry()
         {
             try
             {
-                var reg = Registry.CurrentUser.OpenSubKey(LegacySettingsSubKey, false);
-                if (reg == null) return;
-                IndentEntireProcedureBody = GetSmartIndenterBoolean(reg, "IndentProc", IndentEntireProcedureBody);
-                IndentFirstCommentBlock = GetSmartIndenterBoolean(reg, "IndentFirst", IndentFirstCommentBlock);
-                IndentFirstDeclarationBlock = GetSmartIndenterBoolean(reg, "IndentDim", IndentFirstDeclarationBlock);                
-                AlignCommentsWithCode = GetSmartIndenterBoolean(reg, "IndentCmt", AlignCommentsWithCode);
-                AlignContinuations = GetSmartIndenterBoolean(reg, "AlignContinued", AlignContinuations);
-                IgnoreOperatorsInContinuations = GetSmartIndenterBoolean(reg, "AlignIgnoreOps", IgnoreOperatorsInContinuations);
-                IndentCase = GetSmartIndenterBoolean(reg, "IndentCase", IndentCase);
-                ForceDebugStatementsInColumn1 = GetSmartIndenterBoolean(reg, "DebugCol1", ForceDebugStatementsInColumn1);
-                ForceDebugPrintInColumn1 = ForceDebugStatementsInColumn1;
-                ForceDebugAssertInColumn1 = ForceDebugStatementsInColumn1;
-                ForceStopInColumn1 = ForceDebugStatementsInColumn1;
-                ForceCompilerDirectivesInColumn1 = GetSmartIndenterBoolean(reg, "CompilerCol1", ForceCompilerDirectivesInColumn1);
-                IndentCompilerDirectives = GetSmartIndenterBoolean(reg, "IndentCompiler", IndentCompilerDirectives);
-                AlignDims = GetSmartIndenterBoolean(reg, "AlignDim", AlignDims);
-                AlignDimColumn = GetSmartIndenterNumeric(reg, "AlignDimCol", AlignDimColumn, MaximumAlignDimColumn);
-
-                var eolSytle = reg.GetValue("EOLComments") as string;
-                if (!string.IsNullOrEmpty(eolSytle))
+                using (var reg = Registry.CurrentUser.OpenSubKey(LegacySettingsSubKey, false))
```

Comment:
`OpenSubKey` can return null - this would be better as an explicit call to `Dispose`.